cirq/google/calibration/phased_fsim.py

```diff
@@ -215,8 +252,31 @@ class FloquetPhasedFSimCalibrationResult(PhasedFSimCalibrationResult):
 
     options: FloquetPhasedFSimCalibrationOptions
 
+    @classmethod
+    def _from_json_dict_(
+        cls,
+        parameters: List[Dict[str, Any]],
+        gate: Gate,
+        **kwargs,
+    ) -> 'PhasedFSimCalibrationResult':
+        """Magic method for the JSON serialization protocol.
+
+        Converts serialized dictionary into a dict suitable for
+        class instantiation."""
+        instantiation_dict = {}
+        for entry in parameters:
+            key = (entry['qubit_a'], entry['qubit_b'])
+            instantiation_dict[key] = entry['characterization']
```

Comment:
Maybe a static private helper method on the super class for this, to contain the logic of serialization/deserialization of paraeters in parent??

Reply:
Added.